whois-api/src/test/java/net/ripe/db/whois/api/rest/TrustedQueryTestIntegration.java

```diff
@@ -85,21 +86,6 @@ public void setup() {
 
     // inverse lookup on sponsoring org attribute
 
-    @Test
-    public void inverse_lookup_sponsoring_org_from_untrusted_range_returns_empty() {
```

Comment:
This change belongs in the other PR, but I guess it'll disappear if you merge that one first

Reply:
check the description
Depends on https://github.com/RIPE-NCC/whois/pull/1431

This will disappear from here once the other PR is merged